### PR TITLE
Bug: Force dbt to run single-threaded.

### PR DESCRIPTION
**Pull request by ericf-firebolt**
Resolves #

### Checklist

- [x] I have run this code in development and it appears to resolve the stated issue.
- [x] This PR includes tests, or tests are not required/relevant for this PR.
- [x] I have updated `CHANGELOG.md` and added information about my change.
- [x] If this PR requires a new PyPI release I have bumped the version number.
- [x] I have pulled/merged from the main branch if there are merge conflicts.
- [x] I have verified that this PR contains only code changes relevant to this PR.

### CODE REVIEW

File: dbt/adapters/firebolt/connections.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ class FireboltConnectionManager(SQLConnectionManager):
     def open(cls, connection):
         if connection.state == 'open':
             return connection
-        credentials = cls.get_credentials(connection.credentials)
+        credentials = connection.credentials
```
Comment (swanderz) [inline dbt/adapters/firebolt/connections.py:80]: what's the reason for this change?

File: dbt/include/firebolt/macros/adapters.sql [hunk 2]
```diff
@@ -23,6 +23,9 @@
 
 
 {% macro firebolt__list_schemas(database) %}
```
Comment (swanderz) [inline dbt/include/firebolt/macros/adapters.sql:25]: I like it! clever hack given that list_schemas is invoked with every run pretty much.... does it still work when threads=1?

**Review by kevinmarr — APPROVED** [review]
lgtm

File: dbt/adapters/firebolt/connections.py (hunk 1 above)
Comment (ericf-firebolt) [reply dbt/adapters/firebolt/connections.py:80]: Because `get_credentials()` is

```
def get_credentials(cls, credentials):
        return credentials
        ```
        
Seemed pointless to me. In fact, I ought to remove `get_credentials()`, as it's not in use anymore.

File: dbt/include/firebolt/macros/adapters.sql (hunk 2 above)
Comment (ericf-firebolt) [reply dbt/include/firebolt/macros/adapters.sql:25]: Yep, works great. Thank you @jerco!
